test/utilities/outputView.ts

```diff
@@ -103,3 +98,42 @@ export async function attemptToFindOutputPanelText(
 
   return undefined;
 }
+
+export async function getOperationTime(outputText: string): Promise<string> {
+  const initialTime = outputText.substring(
```

Comment:
When are we considering the initial and end time of operation here?

Reply:
it's in the output text

Reply:
So we are using the cli timestamp to account for operation time.